injector/network_disruption.go

```diff
@@ -369,7 +369,12 @@ func (i *networkDisruptionInjector) applyOperations() error {
 	if len(i.spec.Hosts) == 0 && len(i.spec.Services) == 0 {
 		_, nullIP, _ := net.ParseCIDR("0.0.0.0/0")
 
-		if err := i.config.TrafficController.AddFilter(interfaces, "1:0", i.getNewPriority(), 0, nil, nullIP, 0, 0, "", "1:4"); err != nil {
+		prio, err := i.getNewPriority()
```

Comment:
I'm a bit annoyed by all those `getNewPriority()` calls now done outside of the `AddFilter` function because of the returned error. It duplicates those 4 LOC a bit everywhere. Should this logic be included into the `AddFilter` function itself since it is totally tc related?